cluster/schema/schema.go

```diff
@@ -414,9 +422,10 @@ func makeTenant(name, status string) models.Tenant {
 }
 
 // MakeTenantWithDataVersion creates a tenant with the given name, status, and data version
-func MakeTenantWithDataVersion(name, status string, dataVersion int64) *models.TenantResponse {
+func MakeTenantWithDataVersion(name, status string, belongsToNodes []string, dataVersion int64) *models.TenantResponse {
 	return &models.TenantResponse{
-		Tenant:      makeTenant(name, status),
-		DataVersion: &dataVersion,
+		Tenant:         makeTenant(name, status),
+		DataVersion:    &dataVersion,
+		BelongsToNodes: belongsToNodes,
```

Comment:
nit: `BelongsToNode` -> `OwnedNodes`. Feel bit more accurate to me.

Reply:
I'm reusing the same nomenclature that is used everywhere else in the sharding state. And here I would value consistency over different naming :)